src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -56,16 +60,23 @@ class MediaSelectionContentType extends ComplexContentType implements ContentTyp
      */
     private $permissions;
 
+    /**
+     * @var PropertyMetadataMinMaxValueResolver|null
+     */
+    private $propertyMetadataMinMaxValueResolver;
+
     public function __construct(
         MediaManagerInterface $mediaManager,
         ReferenceStoreInterface $referenceStore,
         RequestAnalyzerInterface $requestAnalyzer = null,
-        $permissions = null
+        $permissions = null,
+        PropertyMetadataMinMaxValueResolver $propertyMetadataMinMaxValueResolver = null
     ) {
         $this->mediaManager = $mediaManager;
         $this->referenceStore = $referenceStore;
         $this->requestAnalyzer = $requestAnalyzer;
         $this->permissions = $permissions;
+        $this->propertyMetadataMinMaxValueResolver = $propertyMetadataMinMaxValueResolver;
```

Comment:
Shouldn't a deprecation be triggered if the given value is null?

Reply:
Since the `PropertyMetadataMinMaxValueResolver` is registered in the `AdminBundle`, it's only available in the admin context. Therefore this argument will always be `null` in the website context, so I can't trigger a deprecation unless I would also inject the context to check

Reply:
Ok, makes sense.